### ISSUE TITLE
Check align mode

### ISSUE THREAD

**Issue by FabriceSalvaire** [FabriceSalvaire/CodeReview#14]
Should add padding to make blocks having the same height.

**Comment by FabriceSalvaire** [comment]
```
    def __init__(cls, class_name, base_classes, attributes):

        type.__init__(cls, class_name, base_classes, attributes)
```

vs

```
    def __init__(cls, class_name, base_classes, namespace):

        print(cls, class_name, base_classes, namespace)

        type.__init__(cls, class_name, base_classes, namespace)
```

4 / 6 lines

**Missing padding line**

**Comment by FabriceSalvaire** [comment]
fixed in b01968a53bc71641dc579fb6af14248e8799e629
